src/main/scala/is/hail/utils/package.scala

```diff
@@ -473,4 +473,21 @@ package object utils extends Logging
     }
     expansion
   }
+
+  def loadFromResource[T](file: String)(reader: (InputStream) => T): T = {
+    val resourceStream = Thread.currentThread().getContextClassLoader.getResourceAsStream(file)
+    assert(resourceStream != null, s"Error while locating file `$file'")
+
+    try {
```

Comment:
I think this code is correct as is. The real error messages are still being printed -- they are prefixed with the Hail message. We need to close the resource in a "finally" block. For example, if `reader` throws an exception, we still want to close the resource.